### DISCUSSION TITLE
Welcome to IndustrialCraft2 Discussions!

### DISCUSSION THREAD

**Discussion by AbbottMc** [IndustrialCraftMC/IndustrialCraft2#1] [【公告】Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SoraChan-uwu** [comment]
I hope it's new IC² will be released in 2024. Haha new IC² will be released before GTA 6

**Reply by SoraChan-uwu** [reply]
I understand that the project is frozen and will not be released in the near future

**Reply by AbbottMc** [reply]
Nope, it will definitely be released before GTA6 😉
